test/ibmq/test_experiment.py

```diff
@@ -291,6 +291,56 @@ def test_experiments_with_public_filters_error(self):
                 'exclude_public and public_only cannot both be True'):
             self.provider.experiment.experiments(exclude_public=True, public_only=True)
 
+    def test_experiments_with_exclude_mine(self):
+        """Tests retrieving experiments with exclude_mine filter."""
+        # Note that we cannot specify the owner when creating the experiment, the value comes
+        # from the user profile via the token so we would have to use different test accounts
+        # to explicitly create separately-owned epxeriments. We should be able to assume that
+        # there is at least one experiment owned by another user in the integration test
+        # environment though.
+        my_exp = self._create_experiment()
+        not_my_experiments = self.provider.experiment.experiments(exclude_mine=True)
+        # The experiment we just created should not be in the set.
+        not_mine_experiment_uuids = []
+        for experiment in not_my_experiments:
+            self.assertNotEqual(
+                experiment.owner, my_exp.owner,
```

Comment:
Need to expose `owner` attribute on the Experiment class:

>test/ibmq/test_experiment.py:307:34: E1101: Instance of 'Experiment' has no 'owner' member (no-member)

Reply:
Actually I'm not sure why this is complaining:

https://github.com/Qiskit/qiskit-ibmq-provider/blob/84898f924fb053c1d032efe7917eb9a99bda57e8/qiskit/providers/ibmq/experiment/experiment.py#L179